README.md

```diff
@@ -36,8 +36,9 @@ Clone repository
 
     git clone https://github.com/oat-sa/package-tao.git
     
-Install via composer missing library and extensions
+Install via composer missing library and extensions. Composer version 2 doesn't work so you've to use version 1.
```

Comment:
this is available only for sprint 121, 
for November release we will have composer 2.*

Reply:
fixed